aten/src/ATen/native/nested/cuda/NestedTensorTransformerFunctions.cpp

```diff
@@ -492,16 +533,17 @@ Tensor flash_attention_helper(
   // If we are passing in query, key, value all the same tensors then we have
   // packed them into one tensor and need to slice for flash attention
   Tensor attention =
-      at::_flash_scaled_dot_product_attention(
+      std::get<0>(at::_flash_attention_forward(
```

Comment:
I updated this function but also not really sure if this even used anymore, in fact I would prefer if it wasn't since it feels vestigial.